drivers/display/display_stm32_ltdc.c

```diff
@@ -321,37 +322,21 @@ static int stm32_ltdc_init(const struct device *dev)
 
 	/* Turn on LTDC peripheral clock */
 	err = clock_control_on(DEVICE_DT_GET(STM32_CLOCK_CONTROL_NODE),
-				(clock_control_subsys_t) &config->pclken);
+				(clock_control_subsys_t) &config->pclken[0]);
 	if (err < 0) {
 		LOG_ERR("Could not enable LTDC peripheral clock");
 		return err;
 	}
 
-#if defined(CONFIG_SOC_SERIES_STM32F4X)
```

Comment:
Should we supply a default clock setup in the SOC devicetree for the STM32F4X to avoid breaking this driver for downstream boards that do not currently configure the DSI clock in devicetree?

Reply:
That might be reasonable, however I doubt this has been used much in practice (at least unchanged) as it depends on the rest of the clock configuration being in the default configuration as well

Reply:
Up to you. If you want to remove the clock setup, please put a note in the migration guide describing the devicetree settings users will need to add in order to restore the clock configuration that would previously have been set for the MIPI DSI